Playthrough.py

```diff
@@ -116,13 +113,17 @@ def iter_reachable_locations(self, item_locations):
             child_failed = Playthrough._expand_regions(
                     child_queue, child_regions, validate_child,
                     adult_queue, adult_regions)
-            # If child reached BDoT, we'll have added BDoT for adult.
+            # If we reached BDoT, we'll have added the root region for the other age.
             # We always have to expand again before checking locations,
-            # since we could have collected all in child before running this.
+            # since we could have collected all in the original age before running this.
             if adult_queue:
                 adult_failed.extend(Playthrough._expand_regions(
                         adult_queue, adult_regions, validate_adult,
                         child_failed, child_regions))
+            if child_queue:
```

Comment:
This part isn't actually required: when starting adult, the first _expand_regions call will add Root to the child_queue before the second call; the third call is specific for starting child and gaining Time Travel.

Reply:
Alright, I fixed it. I added it just in case but I see now it's not needed due to the order of _expand_region calls.